src/flick/discover/views.py

```diff
@@ -50,46 +52,50 @@ def get_top_shows_by_type_and_genre(self, show_type, tag_id):
 
     def get_top_shows_by_genre(self, is_movie, is_tv, is_anime, tag_id):
         if is_movie:
-            self.get_top_shows_by_type_and_genre("movie", tag_id)
+            movies = self.get_top_shows_by_type_and_genre("movie", tag_id)
         if is_tv:
-            self.get_top_shows_by_type_and_genre("tv", tag_id)
+            tvs = self.get_top_shows_by_type_and_genre("tv", tag_id)
         if is_anime:
-            self.get_shows_by_type(
+            animes = self.get_shows_by_type(
                 "anime", "top", ShowAPI.get_top_show_info
             )  # genre does not seem to be fully integrated in the anime api
+        return movies, tvs, animes
 
     def get(self, request, *args, **kwargs):
-        tag_id = request.query_params.get("tag_id", "")
-        print(f"genre: {tag_id}")
-        is_anime = bool(request.query_params.get("is_anime", False))
-        print(f"is_anime: {is_anime}")
-        is_movie = bool(request.query_params.get("is_movie", False))
-        print(f"is_movie: {is_movie}")
-        is_tv = bool(request.query_params.get("is_tv", False))
-        print(f"is_tv: {is_tv}")
-        is_top = bool(request.query_params.get("is_top", False))
-        print(f"is_top: {is_top}")
-        is_now = bool(request.query_params.get("is_now", False))
-        print(f"is_now: {is_now}")
-        is_popular = bool(request.query_params.get("is_popular", False))
-        print(f"is_popular: {is_popular}")
+        # tag_id = request.query_params.get("tag_id", "")
+        # print(f"genre: {tag_id}")
+        # is_anime = bool(request.query_params.get("is_anime", False))
+        # print(f"is_anime: {is_anime}")
+        # is_movie = bool(request.query_params.get("is_movie", False))
+        # print(f"is_movie: {is_movie}")
+        # is_tv = bool(request.query_params.get("is_tv", False))
+        # print(f"is_tv: {is_tv}")
+        # is_top = bool(request.query_params.get("is_top", False))
+        # print(f"is_top: {is_top}")
+        # is_now = bool(request.query_params.get("is_now", False))
+        # print(f"is_now: {is_now}")
+        # is_popular = bool(request.query_params.get("is_popular", False))
+        # print(f"is_popular: {is_popular}")
 
-        self.shows = []
+        # self.shows = []
 
-        if tag_id:
-            self.get_top_shows_by_genre(is_movie, is_tv, is_anime, tag_id)
-        elif is_now:
-            self.get_now_playing_shows(is_movie, is_tv, is_anime)
-        elif is_popular:
-            self.get_popular_shows(is_movie, is_tv, is_anime)
-        elif is_top:
-            self.get_top_shows(is_movie, is_tv, is_anime)
+        # if tag_id:
+        #     self.get_top_shows_by_genre(is_movie, is_tv, is_anime, tag_id)
+        # elif is_now:
+        #     self.get_now_playing_shows(is_movie, is_tv, is_anime)
+        # elif is_popular:
+        #     self.get_popular_shows(is_movie, is_tv, is_anime)
+        # elif is_top:
+        #     self.get_top_shows(is_movie, is_tv, is_anime)
```

Comment:
commented all this out cuz i figured we might wanna add some parts back later
also left all the other helpers there

Reply:
loool sounds good